content/docs/dvclive/api-reference/init.md

```diff
@@ -0,0 +1,61 @@
+# dvclive.init()
+
+Initializes a DVCLive logger.
+
+```py
+def init(
+    path: str = None,
+    resume: bool = False,
+    step: int = 0,
+    summary: bool = True,
+    html: bool = True,
+)
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.init()
+```
+
+## Description
+
+It's usage is optional and focused on configuring the behavior of subsequent
+calls to [`dvclive.log`](/doc/dvclive/api-reference/log) and
+[`dvclive.next_step`](/doc/dvclive/api-reference/next_step).
+
+⚠️ If `path` already exists when this functions is called, it's contents will be
+removed.
+
+## Parameters
+
+- `path` (`dvclive` by default) - The _metrics logs_ (generated by
+  [`dvclive.log`](/doc/dvclive/api-reference/log)) will be saved in separated
+  `.tsv` files under `path` and the _metrics summary_ (generated by
+  [`dvclive.next_step`](/doc/dvclive/api-reference/next_step)) will be saved in
+  a single `{path}.json` file.
+
+- `step` (`0` by default) - the `step` values in log files will start
```

Comment:
do we describe somewhere the format? would be nice to have a link to it

Reply:
Added